dll/win32/shlwapi/ordinal.c

```diff
@@ -4188,6 +4188,18 @@ BOOL WINAPI IsOS(DWORD feature)
         FIXME("(OS_TABLETPC) What should we return here?\n");
         return FALSE;
     case OS_SERVERADMINUI:
+#ifdef __REACTOS__
+        {
+            DWORD value = FALSE, size = sizeof(value);
```

Comment:
Is this something that should be cached?
I can imagine some shitty written application spamming this every WM_PAINT or whatnot

Reply:
Note that we have the `GetVersionExA` call much above that would also be called every time in such scenario too.

Reply:
> Is this something that should be cached?

You can see in Process Monitor that this value is not cached in Windows, even in NT6. I can cache it if you want though...

Reply:
> > Is this something that should be cached?
> 
> You can see in Process Monitor that this value is not cached in Windows, even in NT6. I can cache it if you want though...

No, if it's not needed keep it simple.